src/VirtualClient/VirtualClient.Main/profiles/PERF-MYSQL-SYSBENCH-OLTP.json

```diff
@@ -231,7 +231,7 @@
                 "Action": "SetGlobalVariables",
                 "Benchmark": "OLTP",
                 "DiskFilter": "$.Parameters.DiskFilter",
-                "Variables": "MAX_PREPARED_STMT_COUNT=1000000;MAX_CONNECTIONS=1000000",
+                "Variables": "MAX_PREPARED_STMT_COUNT=1000000;MAX_CONNECTIONS=100000",
```

Comment:
value of the MAX_CONNECTIONS can be set to 100000. If it is set to any larger value, it will get value of 100000.